### DISCUSSION TITLE
commafeed.com server upgrade

### DISCUSSION THREAD

**Discussion by Athou** [Athou/commafeed#1127] [Announcements]
As you probably have noticed, performances of commafeed.com have been degrading slowly for the past few years, even more so since the release of v3.0, to the point where it is now almost unusable.

The server has been in service since 2013, has very slow drives and they seem to get even slower.
There also have been optimizations to the feed refresh engine, feeds are now refreshing faster, but at a cost.

To fix this (and as discussed [here](https://github.com/Athou/commafeed/issues/1125) and [here](https://github.com/Athou/commafeed/discussions/1015#discussioncomment-6522738)), I’m going to move commafeed.com to a newer server with SSD (NVMe) drives.
Users, settings and feeds will be migrated. However, entries (and thus read/unread status, tags and stars) will not. The drives are too slow and it would take weeks to migrate the 300 million existing feed entries 🙁
The vast majority of feed entries are read once and never starred/tagged, I think this is an acceptable trade-off in order to get good performance back. I’ve been playing with the new server for a bit and it’s way faster than the old one (at least 20x), I think you’ll be pleasantly surprised by how reactive the app feels.

As a side note, commafeed.com runs on a server that costs around 350€ per year. I managed to find a new server that will cost about the same. While not too expensive, it still comes from my own pocket, as donations only cover about 20% of this cost. CommaFeed has always been a passion project of mine and I never wanted to add paid accounts but if you like CommaFeed and can afford it, please consider a [donation](https://github.com/sponsors/Athou) 🙂

**Comment by hcirlu** [comment]
Donated. Hope others will follow.

**Comment by michael-wallace-bv** [comment]
The difference is immediately obvious 👍 Great job! Definitely worth a donation.

**Reply by k2oly** [reply]
AGREED! the upgrade made things so slow, i was planning to switch to a different reader. the alternatives are awful, but i chose one & planned to switch the day that Jeremie posted this. THANK YOU JEREMIE!

**Comment by harnessg** [comment]
I started tagging posts a few months ago. Is there any way for me to save those posts or download the content? Well, now it appears that all of my starred and tagged content is gone. Are you doing the server move now?

**Reply by Athou** [reply]
The old server is still running, do you want me to make it accessible again on another URL to give you time to retrieve what you need?

**Reply by hcirlu** [reply]
would be appreciated.

**Reply by Athou** [reply]
The old server is now back online at https://old.commafeed.com

It will become offline at the end of the month, because I stopped paying for it 🙂

**Reply by harnessg** [reply]
Thanks. That's a big help.

**Comment by ingokegel** [comment]
For anyone who can afford it, It should really be a no-brainer to donate a couple of $ per month for this excellent service. Performance on the new server is amazing 👍‍

**Comment by CasimirBergamotte** [comment]
The difference is so amazing!
Just a question: in my thread, there are blogs hosted by free platforms [Canalblog](https://www.canalblog.com/) and [Over-blog](https://www.over-blog.com/) and with the new server, for those I keep having messages such as "Unable to refresh feed : xxxxxxx.over-blog.com: Temporary failure in name resolution" or "Unable to refresh feed : xxxxxxxx.canalblog.com: Temporary failure in name resolution" (xxxx here replacing the names of the blogs for the example). 
Is there anything I have to fix (and how to do it) to get fresh alerts like I used to get with the former server?
Thank you to anyone who can help me with that.

**Reply by Athou** [reply]
I made a small change to the DNS resolver, does it work now?

**Reply by CasimirBergamotte** [reply]
Thank you for your help but it does not seem like it works... I cleared the browser cache, deconnected and reconnected from CommaFeed but the feeds still appear in red and the same error messages show up.

**Reply by Athou** [reply]
It should work the next time CommaFeed refreshes those feed, or you can manually requeue your feeds from the profile menu ("Fetch all my feeds now").

**Reply by CasimirBergamotte** [reply]
OK. Anyway, thank you for your help ans your work.

**Reply by CasimirBergamotte** [reply]
It does work! Everything's back to normal. Thank you so much!

**Comment by tdubbf** [comment]
Hi, my account or settings were migrated so I created a new. Is there a way to retrieve my old feeds?
